### PR TITLE
chore(ap): Merge new ADS admin functionality

### PR DESCRIPTION
**Pull request by eirikurn**
## What

This brings the admin portal ADS module up to date with the previous ADS admin UI.

## Why

So we can deprecate the old and release the new.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/portals/admin/air-discount-scheme/src/screens/CreateDiscount/CreateDiscount.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  return (
+    <>
+      <GridContainer>
+        <GridRow>
+          <GridColumn span={['12/12', '12/12', '7/12', '8/12', '9/12']}>
+            {discountCode.length ? (
+              <>
+                <h1>{discountCode}</h1>
+                <p>
+                  Umsýsluviðmótið geymir þessa kóða ekki. Þeir munu birtast í
```
Comment (kksteini) [inline libs/portals/admin/air-discount-scheme/src/screens/CreateDiscount/CreateDiscount.tsx:34]: Thanks for porting this.
Should we add localization?

**Review by kksteini — APPROVED** [review]
LGTM

File: libs/portals/admin/air-discount-scheme/src/screens/CreateDiscount/CreateDiscount.tsx (hunk 1 above)
Comment (eirikurn) [reply libs/portals/admin/air-discount-scheme/src/screens/CreateDiscount/CreateDiscount.tsx:34]: It's not a priority at the moment for admin systems

**Review by snaerseljan — APPROVED** [review]
LGTM! Good job 🥇

**Pull request merged**